src/main/java/org/example/App.java

```diff
@@ -1,13 +1,72 @@
 package org.example;
 
+import org.example.dao.*;
+import org.example.model.*;
+import org.example.service.*;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Properties;
+
 /**
  * Hello world!
  *
  */
 public class App 
 {
+
     public static void main( String[] args )
     {
-        System.out.println( "Hello World!" );
+        Connection connection;
+        try {
+            Properties properties = new Properties();
+            properties.load(Files.newInputStream(Paths.get("src/main/resources/db.properties")));
+
+            String url = properties.getProperty("db.url");
+            String username = properties.getProperty("db.user");
+            String password = properties.getProperty("db.password");
+
+            connection = DriverManager.getConnection(url, username, password);
+            UserService userService = new UserService(new UserDAO(connection));
+            userService.getAllUsers();
+
+            User jonathan = new User("jt", "jt@gmail.com", "password", LocalDate.now(), null, null);
+//            userService.createUser(jonathan);
```

Comment:
comments in code

Reply:
comments came from different branch and since have been taken out